monkey/monkey_island/cc/server_utils/encryptor.py

```diff
@@ -51,7 +51,17 @@ def dec(self, enc_message):
         enc_message = base64.b64decode(enc_message)
         cipher_iv = enc_message[0 : AES.block_size]
         cipher = AES.new(self._cipher_key, AES.MODE_CBC, cipher_iv)
-        return self._unpad(cipher.decrypt(enc_message[AES.block_size :]).decode())
+        try:
+            dec_message = self._unpad(cipher.decrypt(enc_message[AES.block_size :]).decode())
+            return dec_message
+        except UnicodeDecodeError:
+            print("monkey-island secret key does not match MongoDB encrypted key")
+            print("Possible solutions:")
+            print("Run'docker container start -a monkey-island'")
+            print("Kill and restart the MongoDB container.")
+        except Exception as e:
+            print(e)
+        return None
```

Comment:
The other code that calls this isn't expecting to receive `None`. I'm not sure how all of that code will behave under these circumstances. Monkey Island should probably exit if it encounters this exception. We should catch the `UnicodeDecodeError`, and re-raise a new, custom exception. Something like `DecryptionError`.

Ultimately, this exception should result in Monkey Island exiting. The user may not be looking at the log and may not realize there's a problem. It's not a good idea for us to run if we can't properly read mongodb.